Makefile

```diff
@@ -29,35 +34,43 @@ check-format: format
 vet:    ## Run go vet on source code.
 	go vet $(ALLPKGS)
 
+
 # Build BPF
-CLANG := clang
 CLANG_INCLUDE := -I../../..
 BPF_CFLAGS := -g -O2 -Wall -fpie -target bpf -DCORE -D__BPF_TRACING__ -D__TARGET_ARCH_$(ARCH) 
 TARGETS := \
-		  test-data/tc.ingress \
-		  test-data/tc \
-		  test-data/test.map \
-		  test-data/test_license \
-		  test-data/invalid_map \
-		  test-data/recoverydata \
-		  test-data/test-kprobe \
-		  test-data/xdp \
-		  test-data/ring_buffer
+		  $(TESTDATADIR)/tc.ingress \
+		  $(TESTDATADIR)/tc \
+		  $(TESTDATADIR)/test.map \
+		  $(TESTDATADIR)/test_license \
+		  $(TESTDATADIR)/invalid_map \
+		  $(TESTDATADIR)/recoverydata \
+		  $(TESTDATADIR)/test-kprobe \
+		  $(TESTDATADIR)/xdp \
+		  $(TESTDATADIR)/ring_buffer
 
 %.bpf.elf: %.bpf.c
 	$(CLANG) $(CLANG_INCLUDE) $(BPF_CFLAGS) -c $< -o $@
 
-vmlinuxh:
-	bpftool btf dump file /sys/kernel/btf/vmlinux format c > $(abspath ./test-data/vmlinux.h)
+## check if the vmlinux exists in /sys/kernel/btf directory
+VMLINUX_BTF ?= $(wildcard /sys/kernel/btf/vmlinux)
+ifeq ($(VMLINUX_BTF),)
+$(error Cannot find a vmlinux)
+endif
+
+$(TESTDATADIR)/vmlinux.h:
+	$(BPFTOOL) btf dump file $(VMLINUX_BTF) format c > $@
 
+LOGFILE_PATH ?= stdout
```

Comment:
Minor nit: Can we move this to the top? After line 8 maybe?

Reply:
fixed to move that to L9